lsan_arangodb_suppressions.txt

```diff
@@ -17,3 +17,7 @@ leak:snowball
 #  4 in v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) 
 # SUMMARY: AddressSanitizer: 139 byte(s) leaked in 2 allocation(s).
 leak:V8Buffer::New
+
+# Suppressed to see other issue
+# TODO(mpoeter) ?
+leak:LogContext::Current::appendEntry
```

Comment:
Not necessary in 3.9. replication2 code doesn't exist in this branch.

Reply:
But it's from RestServer...?

Reply:
Anyway, it doesn't make it worse

Reply:
LogContext is from replication2, isn't it?

Reply:
https://github.com/arangodb/arangodb/blob/3.10/lib/Logger/LogContext.h

No, it's other log, some Manuel experiment with structured logging if I remember correct

Reply:
Ah, you are right.